payments-core/src/main/java/com/stripe/android/ConfirmStripeIntentParamsFactory.kt

```diff
@@ -21,6 +21,7 @@ sealed class ConfirmStripeIntentParamsFactory<out T : ConfirmStripeIntentParams>
 
     abstract fun create(
         createParams: PaymentMethodCreateParams,
+        confirmPaymentMethodOptions: PaymentMethodOptionsParams? = null,
```

Comment:
Nit: Can you rename this to `optionsParams` and make sure that the subclasses have the same argument names?